canopy/app/src/loadSaplings.js

```diff
@@ -74,11 +90,13 @@ export async function loadAllSaplings() {
     getConfigSaplings()
   ]);
 
-  const configSaplingsAreLoaded = await loadConfigSaplings(
-    configSaplingResponse
-  );
+  const configSaplingsAreLoaded =
+    (await loadConfigSaplings(configSaplingResponse)) &&
```

Comment:
This doesn't maximize concurrency.

You could `await Promise.all([loadConfigSaplings(configSaplingResponse), loadSaplingStyles(configSaplingResponse)])

Otherwise they'll happen one after the other with await.

Reply:
updated